index.js

```diff
@@ -3638,15 +3638,21 @@
   //.
   //. > S.get(S.is(Number), 'x', {})
   //. Nothing
+  //.
```

Comment:
The existing example are great.  But maybe a little conservative/sensible.  I think we can pre-emptively answer some questions about the behaviour of `Any` in conjunction with the `Accessible` constraint by demonstrating a completely non sensical input both within and outside an object.  Perhaps something like:

```js
S.get( S.is(Number), 'x', { x: 'purple' } ) 
//=> Nothing

S.get( S.is(Number), 'x', 'purple' ) 
// => Type Error
// value 'purple' does not satisfy the Accessible type class.
```